maintainers-guide.md

```diff
@@ -168,27 +168,35 @@ organization](https://github.com/bazelbuild) to be able to add or remove labels.
 The following definitions for priority will be used by the maintainers to triage
 issues.
 
-* **P0** - Major broken functionality that causes a Bazel release (minus release
-  candidates) to be unusable, or a downed service that severely impacts
-  development of the Bazel project. This includes regressions introduced in a
-  new release that blocks a significant number of users, or an incompatible
-  breaking change that was not compliant to the [Breaking
+* [**P0**](https://github.com/bazelbuild/bazel/labels/P0) - Major broken
+  functionality that causes a Bazel release (minus release candidates) to be
+  unusable, or a downed service that severely impacts development of the Bazel
+  project. This includes regressions introduced in a new release that blocks a
+  significant number of users, or an incompatible breaking change that was not
+  compliant to the [Breaking
   Change](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/edit?pli=1#heading=h.ceof6vpkb3ik)
   policy. No practical workaround exists.
-* **P1** - Critical defect or feature which should be addressed in the next
-  release, or a serious issue that impacts many users (including the development
-  of the Bazel project), but a practical workaround exists. Typically does not
-  require immediate action. In high demand and planned in the current quarter's
-  roadmap.
-* **P2** - Defect or feature that is tied to roadmaps or upcoming releases.
-  Moderate live issue in a released Bazel version that is inconvenient for a
-  user that needs to be addressed in an upcoming release and/or a easy workaround
-  exists.
-* **P3** - Desirable minor bug fix or enhancement with small impact. Not
-  prioritized into Bazel roadmaps or any imminent release. May never be fixed.
-* **P4** - Low priority defect or feature request that is unlikely to get
-  closed. Can also be kept open for a potential re-prioritization if more users
-  are impacted.
+* [**P1**](https://github.com/bazelbuild/bazel/labels/P1) - Critical defect or
+  feature which should be addressed in the next release, or a serious issue that
+  impacts many users (including the development of the Bazel project), but a
+  practical workaround exists. Typically does not require immediate action. In
+  high demand and planned in the current quarter's roadmap.
+* [**P2**](https://github.com/bazelbuild/bazel/labels/P2) - Defect or feature
+  that is tied to roadmaps or upcoming releases.  Moderate live issue in a
+  released Bazel version that is inconvenient for a user that needs to be
+  addressed in an upcoming release and/or a easy workaround exists.
+* [**P3**](https://github.com/bazelbuild/bazel/labels/P3) - Desirable minor bug
+  fix or enhancement with small impact. Not prioritized into Bazel roadmaps or
+  any imminent release. May never be fixed.
+* [**P4**](https://github.com/bazelbuild/bazel/labels/P4) - Low priority defect
```

Comment:
Should this be removed? The description makes it sound like the ice box too. (And realistically, I believe that's how P4s have been used.)

Reply:
Perhaps, but at the moment I don't want to go through all P4s to judge whether they are better placed in the icebox or marked as P3 with help wanted.

Reply:
I don't think it is the same as ice-box.  Ice-box has the very agile connotations to me. It means things that are important, but we are not ready yet. P4 is far less important. We could have an P1+icebox task. It would be something that we know we must work on as soon as possible, but something is blocking us starting on that right now.

Reply:
The whole point of this change, I believe, is to have a mechanism to archive bugs that we know are somehow valid but think they'll never be addressed -- and do so in a way that doesn't sound like "closed, we don't care".

If icebox has different connotations, we could pick a different name. But I'd avoid adding more complexity to the process with things like p1+icebox.

Re. the P4-P3 thing. We could "simply" mass-edit all P4 bugs and say: "Hey, we are moving this to the new icebox label because we think it better reflects how we've been treating P4s until now. If you disagree, please reply and we'll update the labels." There are 100 of them.

Reply:
I agree with Julio's reasoning. That was the intention of the icebox label.

Reply:
Not changing anything wrt P4s for now.